src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -5034,7 +5085,7 @@ sig_handler(int code, siginfo_t *siginfo, void *ctx)
 	int                     rc, fd, idx_map;
 	d_iov_t                 iov;
 	d_sg_list_t             sgl;
-	struct ucontext_t      *context = (struct ucontext_t *)ctx;
+	ucontext_t             *context = (ucontext_t *)ctx;
```

Comment:
Technically you don't need this cast here as void * aliases to any pointer but this is a very common mistake.
```suggestion
	ucontext_t             *context = ctx;
```